### PR TITLE
Fonts, Toast service and Story 738,736,714 implemented

### PR DESCRIPTION
**Pull request by vishwanath1004**

### CODE REVIEW

File: src/app/shared/components/session-card/session-card.component.scss [hunk 1]
```diff
@@ -39,4 +39,10 @@
     margin-top: -15px;
     padding: 0px 35px 5px 0px;
     color: var(--ion-color-primary);
-    }
\ No newline at end of file
+    }
+.ellipsis{
```
Comment (kiranharidas187) [inline src/app/shared/components/session-card/session-card.component.scss:43]: Please put this class ina common place and reuse

File: src/app/shared/components/session-sqr-card/session-sqr-card.component.scss [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
+    padding: 12px 5px;
+    min-height: 180px;
+    max-height: 190px;
+    border-radius: 10px;
     .btn-width-small{
         min-width: 50%;
     }
+}
+
+.ellipsis{
```
Comment (kiranharidas187) [inline src/app/shared/components/session-sqr-card/session-sqr-card.component.scss:12]: reuse

File: src/app/shared/components/session-sqr-card/session-sqr-card.component.scss [hunk 3]
```diff
@@ -1,7 +1,17 @@
 .session-sqr-card{
     text-align: center;
-    padding: 12px 2px;
+    padding: 12px 5px;
+    min-height: 180px;
+    max-height: 190px;
```
Comment (kiranharidas187) [inline src/app/shared/components/session-sqr-card/session-sqr-card.component.scss:5]: why do we need min and max width

File: src/assets/fonts/noto_Sans/OFL.txt [hunk 4]
```diff
@@ -0,0 +1,93 @@
+Copyright 2012 Google Inc. All Rights Reserved.
```
Comment (kiranharidas187) [inline src/assets/fonts/noto_Sans/OFL.txt:1]: rm

File: src/global.scss [hunk 5]
```diff
@@ -43,8 +45,19 @@
   flex-direction: column;
 }
 
+h5 , h4, h3 {
```
Comment (kiranharidas187) [inline src/global.scss:48]: Why are we modifying this default tags??

File: src/theme/variables.scss [hunk 6]
```diff
@@ -234,3 +234,32 @@
     --ion-card-background: #1e1e1e;
   }
 }
+
+// Mentoring app fonts
+@font-face {
+  font-family: 'NotoSans';
```
Comment (kiranharidas187) [inline src/theme/variables.scss:240]: This should be different for different fonts,

Comment (kiranharidas187) [reply src/theme/variables.scss:240]: NotoSans-Bold

File: src/app/app.component.ts [hunk 7]
```diff
@@ -45,7 +47,7 @@ export class AppComponent {
   }
 
   setLanguage(lang){
-    this.localStorage.setLocalData(localKeys.selectedLanguage,'en').then(data =>{
+    this.localStorage.setLocalData(localKeys.selectedLanguage,'lang').then(data =>{
```
[the remark this answers is not in the document]
Comment (vishwanath1004) [reply src/app/app.component.ts:50]: before it was hard code with 'en', now i made as parameterised

**Pull request merged**